buffer/src/main/java/io/netty/buffer/PoolSubpage.java

```diff
@@ -25,6 +25,7 @@
 final class PoolSubpage<T> implements PoolSubpageMetric {
 
     final PoolChunk<T> chunk;
+    final int elemSize;
```

Comment:
strictly speaking this is not required by this change but while I did debug I noticed this can be final so I changed it within this PR